src/styles/_catalog.scss

```diff
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
```

Comment:
I use dev tools and under 1024px wide your grid container stick to bottom with no empty space